src/auto/enums.rs

```diff
@@ -6512,6 +6592,84 @@ impl SetValue for StateType {
     }
 }
 
+#[derive(Debug, Eq, PartialEq, Ord, PartialOrd, Hash, Clone, Copy)]
+#[non_exhaustive]
+pub enum TextBufferTargetInfo {
```

Comment:
This does not seem to be used in any API we have, so just having the enum generated wouldn't provide anything useful.